osu.Game/Online/Leaderboards/Leaderboard.cs

```diff
@@ -150,7 +151,7 @@ protected PlaceholderState PlaceholderState
                         break;
 
                     case PlaceholderState.NotLoggedIn:
-                        replacePlaceholder(new MessagePlaceholder(@"Please sign in to view online leaderboards!"));
+                        replacePlaceholder(new LoginPlaceholder(@"view online leaderboards!"));
```

Comment:
Concating message strings is not friendly for localisation. Leave here for further consideration.

Reply:
Removed the string concatenation from the constructor and put it back like it was before. Should be fine for now